### PR TITLE
Rubify all the things

### PR DESCRIPTION
**Pull request by segiddins**

### CODE REVIEW

File: lib/plugin.rb [hunk 1]
```diff
@@ -1,50 +1,53 @@
...
+          target.label == 'Pods-' + user_options['target'].to_s
         end
-              
+
         if target
-          target.store_pod 'Keys', :path => 'Pods/CocoaPodsKeys/'
+          target.store_pod 'Keys', :path => keys_path.to_path
         else
-          puts "Could not find a target named '#{user_options["target"]}' in your Podfile. Stopping Keys.".red
+          Pod::UI.puts "Could not find a target named '#{user_options['target']}' in your Podfile. Stopping Keys.".red
```
Comment (orta) [inline lib/plugin.rb]: :+1:

File: lib/pod/command/keys/set.rb [hunk 2]
```diff
@@ -40,16 +39,16 @@ def run
           # info "Saving into the keychain."
 
           keyring = current_keyring
-          keyring.keys << @key_name.gsub("-", "_")
+          keyring.keys << @key_name.tr('-', '_')
```
Comment (orta) [inline lib/pod/command/keys/set.rb]: what is tr?

Comment (segiddins) [inline lib/pod/command/keys/set.rb]: it's `translate` so here it `translates` all instance of `-` to `_`

Comment (orta) [inline lib/pod/command/keys/set.rb]: great, I use gsub all the time, tr sounds well better

Comment (segiddins) [inline lib/pod/command/keys/set.rb]: It's simpler (and faster) when you're just changing one character -- `gsub` on the other hand has the full power of regular expressions

**Comment by orta** [comment]
my arch nemesis rubocop ey

**Pull request merged**